src/components/WalletDropdown/index.tsx

```diff
@@ -17,6 +17,9 @@ const WalletWrapper = styled.div`
   top: 60px;
   right: 70px;
   background-color: ${({ theme }) => theme.backgroundSurface};
+  border: ${({ theme }) => `1px solid ${theme.backgroundOutline}`};
+  box-shadow: 0px 24px 32px rgba(0, 0, 0, 0.04), 0px 16px 24px rgba(0, 0, 0, 0.04), 0px 4px 8px rgba(0, 0, 0, 0.04),
```

Comment:
could you confirm with fred about this box-shadow? i feel like we're trying to avoid non color variables recently

Reply:
update on this, it was indeed not correct.  Fred updated to correct and I updated it to match that